src/models/kudu6/README.md

```diff
@@ -37,7 +37,7 @@ The System76 Kudu is a laptop with the following specifications:
 - Power
     - 230W (19.5V, 11.8A) AC adapter
         - Included AC adapter: Chicony A17-230P1A
-        - Included AC adapter uses C5 (Cloverleaf) power cord
+        - Included AC adapter uses C13 ("kettle cord") power cord
```

Comment:
Looks like "kettle plug" rather than "kettle cord" is more popular. Still, Jacob is right about matching what the Bonobo says. "C13" is a search term that's far more likely to yield the correct result anyway, so the nickname isn't all that helpful.